.bazelci/presubmit.yml

```diff
@@ -50,6 +47,12 @@ tasks:
     working_directory: examples
     test_targets:
     - //...
+  docs:
+    name: Docs
+    platform: ubuntu1804
```

Comment:
Does this actually care about the ubuntu version and/or not work on osx?

Reply:
It works on OSX I have developed it on OSX. Added the test for OS X just to make sure we don't break our own development workflow.